src/common/src/constants.rs

```diff
@@ -0,0 +1,16 @@
+pub const WORKING_DIR: &str = "/tmp/tracer/";
+pub const PID_FILE: &str = "/tmp/tracer/tracerd.pid";
+pub const STDOUT_FILE: &str = "/tmp/tracer/tracerd.out";
+pub const STDERR_FILE: &str = "/tmp/tracer/tracerd.err";
+pub const LOG_FILE: &str = "/tmp/tracer/daemon.log";
+pub const FILE_CACHE_DIR: &str = "/tmp/tracer/tracerd_cache";
+pub const DEBUG_LOG: &str = "/tmp/tracer/debug.log";
```

Comment:
also we have a lot of files here, the debug.log usually has the same rows everytime
we have 100 of thousands of rows of nothing
<img width="1076" alt="image" src="https://github.com/user-attachments/assets/5e0f89a8-04fa-4b20-90c9-eea0ff15ea9a" />